.github/workflows/update-changelog.yml

```diff
@@ -0,0 +1,29 @@
+name: Update CHANGELOG
+on:
+  workflow_dispatch
```

Comment:
Should we consider running this on a schedule - perhaps every Tuesday evening before a snap?